docs/source/release/v3.9.0/framework.rst

```diff
@@ -55,6 +55,7 @@ Bug Fixes
 - :ref:`LoadEventNexus <algm-LoadEventNexus>` should no longer leak memory when the execution is cancelled.
 - :ref:`LoadNexusProcessed <algm-LoadNexusProcessed>` will now load the stored workspace names from a processed Nexus file in the case of multiperiod data.
 - If a run is aborted and restarted, the ``running`` log in the workspace will correctly reflect this. (``running`` will be false at all times before the abort.)
+- :ref:`SumSpectra <algm-SumSpectra>` should correctly handle special values in the data such as NaN which previously ended up in the summed spectra.
```

Comment:
Are we sure setting it to 0 is the correct solution? Could it just be a sign that there is a deeper problem with the algorithms using it , that this fix would mask away?